src/scripts/components/PillsInput.jsx

```diff
@@ -37,6 +37,18 @@ class PillsInput extends React.Component {
       ))
     );
 
+    const { inputProps } = this.props;
+
+    const propsForInput = {
+      naked: inputProps.naked || true,
+      onChange: inputProps.onChange || this.handleChange,
+      name: inputProps.name || this.props.name,
```

Comment:
this is the only one I think we don't need. They're effectively the same so I'd go with `inputProps.name` and get rid of `name` as a prop altogether.

Reply:
👍